tmux.conf

```diff
@@ -27,6 +27,12 @@ set -g status-fg '#aaaaaa'
 set -g status-left ''
 set -g status-right ''
 
+# allow ctrl + b + - to split window horizontally
+bind - split-window -v 
+
+# allow ctrl + b + | to split window vertically
+bind | split-window -h
```

Comment:
Not sure if it matters, but I did this on `\` instead of `|` to avoid an extra keypress on a very common operation. tmux 3.x has a breaking change where backslashes need to be escaped in the config file:

```
bind-key \\ split-window -h  -c '#{pane_current_path}'
```